pkg/controller/jobframework/reconciler.go

```diff
@@ -368,7 +369,29 @@ func (r *JobReconciler) equivalentToWorkload(job GenericJob, object client.Objec
 	if owner.Name != object.GetName() {
 		return false
 	}
-	return job.EquivalentToWorkload(*wl)
+
+	jobPodSets := job.PodSets()
+
+	if !workload.CanBePartiallyAdmitted(wl) || job.IsSuspended() {
```

Comment:
```suggestion
	if (features.Enabled(features.PartialAdmission) && !workload.CanBePartiallyAdmitted(wl)) || job.IsSuspended() {
```

Should we check if the feature is enabled?

Reply:
I make better sense to ensure the minCount is not set when the feature is disabled. I'll do it in #917

Reply:
What happens in the following situation If we check the minCount only in the webhook?

1. Start kueue-controller-manager with `PartialAdmission=enabled`.
2. Create a job with `minCount`.
3. Re-start kueue-controller-manager with `PartialAdmission=disabled`.

Reply:
the workload will not be equivalent and will get recreated, which is fine since we no longer accept palatial admission after 3.

Reply:
> which is fine since we no longer accept palatial admission after 3.

Agree.
Maybe, we need to say the specification in the docs.